plugins/auth-backend/api-report.md

```diff
@@ -247,6 +247,13 @@ export const createOAuth2Provider: (
   options?: OAuth2ProviderOptions | undefined,
 ) => AuthProviderFactory;
 
+// Warning: (ae-missing-release-tag) "createOauth2ProxyProvider" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
Let's make sure that there are no new warnings added for new functionality  :)

Reply:
Done 👍